yorkie/yorkie.go

```diff
@@ -71,6 +86,7 @@ func (r *Yorkie) Shutdown(graceful bool) error {
 		return nil
 	}
 
+	r.metricsServer.Shutdown(graceful)
```

Comment:
It would be nice to stop the user's incoming requests first then close the metrics last.